File: Updater/src/test/java/jk_5/nailed/updater/TestUtils.java
package jk_5.nailed.updater;

import org.junit.Ignore;

/**
 * No description given
 *
 * @author jk-5
 */
@Ignore
public class TestUtils {

    public static String shuffleCase(String input){
        char[] chars = input.toCharArray();
        for(int i = 0; i < chars.length; i += 2){
            chars[i] = Character.toUpperCase(chars[i]);
        }
        return new String(chars);
    }
}

File: Updater/src/test/scala/jk_5/nailed/updater/TestUtils.scala
package jk_5.nailed.updater

import org.junit.Ignore

/**
 * No description given
 *
 * @author jk-5
 */
@Ignore object TestUtils {

  def shuffleCase(input: String): String = {
    val chars: Array[Char] = input.toCharArray
    for(i <- 0 until chars.length){
      if(i % 2 == 0) chars(i) = Character.toUpperCase(chars(i))
    }
    new String(chars)
  }
}
